loader/loader.go

```diff
@@ -242,12 +242,14 @@ func Load(
 }
 
 func resolveUsingLoaders(logger logrus.FieldLogger, name string) (*url.URL, error) {
-	_, loader, loaderArgs := pickLoader(name)
+	loaderName, loader, loaderArgs := pickLoader(name)
 	if loader != nil {
 		urlString, err := loader(logger, name, loaderArgs)
 		if err != nil {
+			logger.Warnf("Using magic urls for `%s` is deprated. Please use normal urls", loaderName)
```

Comment:
```suggestion
			logger.Warnf("Using magic URLs for `%s` is deprecated. Please use normal URLs", loaderName)
```

Though I'm wondering if people will understand what we mean by "magic URLs". We don't use that name anywhere, I think only internally.